src/components/fundamentals/icons/search-icon.tsx

```diff
@@ -1,16 +1,17 @@
-import PropTypes from "prop-types"
 import React from "react"
+import IIconProps from "./icon-interface"
 
-const SearchIcon = props => {
-  const { size, color, ...rest } = props
+const SearchIcon: React.FC<IIconProps> = ({
+  size = "24px",
+  color = "currentColor",
+}) => {
   return (
     <svg
       width={size}
       height={size}
-      viewBox="0 0 24 24"
+      viewBox={`0 0 ${size} ${size}`}
```

Comment:
Small change, don't change the viewbox of Icons, as it will result in the icons becoming cropped. Changing size alone will scale the icons properly. 

Also applies to all other icons